client/src/services/requests.js

```diff
@@ -0,0 +1,14 @@
+const APIKEY = '8f61443bc44ab0f2c23082795b681a12';
```

Comment:
you should avoid sharing API Keys on Github. us .dotenv to prevent this

Reply:
Thanks for the heads up! Already aware of that issue and the Key is already invalidated and replaced locally. Next push will include hidden API.